modules/controlplane/main.tf

```
@@ -254,43 +317,33 @@ resource "aws_ecs_task_definition" "control_plane_task" {
           name  = "CF_EVENT_HANDLER_SQS_QUEUE",
           value = var.sqs_queue_name
         },
-
-        {
-          name  = "CF_PAGERDUTY_CLIENT_ID",
-          value = var.pager_duty_client_id
-        },
-        {
-          name  = "CF_PAGERDUTY_REDIRECT_URL",
-          value = "${var.app_url}/api/v1/oauth2/callback/pagerduty"
-        },
         {
           name  = "CF_FRONTEND_URL",
           value = var.app_url
         },
         {
-          name  = "CF_API_URL",
-          value = var.app_url
+          name  = "CF_MAKE_AVAILABLE_CRON_SCHEDULE",
+          value = "0 */5 * * * *"
         },
         {
-          name  = "CF_AUTHZ_URL",
-          value = var.app_url
+          name  = "CF_MAKE_AVAILABLE_ENABLED",
+          value = "true"
         },
         {
-          name  = "CF_ACCESS_URL",
-          value = var.app_url
+          name  = "CF_OIDC_AUTHORITY_URL",
+          value = var.scim_source
```

Why has the order of all of these changes?
Did it make a difference?
It looks like this has been changed to the wrong value so I suggest that we revert all the environment variable changes in this file

```suggestion
          value = var.scim_source <-- this is incorrect
```